lib/ClangImporter/ClangImporter.cpp

```diff
@@ -730,22 +730,22 @@ importer::addCommonInvocationArguments(
 
   } else if (triple.isOSDarwin()) {
     // Special case CPU based on known deployments:
-    //   - arm64 deploys to cyclone
+    //   - arm64 deploys to apple-a7
     //   - arm64 on macOS
     //   - arm64 for iOS/tvOS/watchOS simulators
-    //   - arm64e deploys to vortex
-    // and arm64e (everywhere) and arm64e macOS defaults to the "vortex" CPU
+    //   - arm64e deploys to apple-a12
+    // and arm64e (everywhere) and arm64e macOS defaults to the "apple-a12" CPU
     // for Darwin, but Clang only detects this if we use -arch.
```

Comment:
Not part of this PR but this comment might need to updated in a later commit. I don't really understand what it is saying.

```
and arm64e (everywhere) and arm64e macOS defaults to the "apple-a12" CPU
```

Not sure what the first "and" is referring to. Maybe it's just superfluous? Or there's some missing indentation?

There's also mention of `-arch` but there are no `-arch` flags below, we are using `-mcpu`.